code/controllers/subsystem/mapping.dm

```diff
@@ -0,0 +1,38 @@
+SUBSYSTEM_DEF(mapping)
+   	name = "Mapping"
+   	init_order = INIT_ORDER_MAPPING // 9
+   	flags = SS_NO_FIRE
+
+/datum/controller/subsystem/mapping/New()
+	NEW_SS_GLOBAL(SSmapping)
```

Comment:
Um. This is making the same SS twice.

Reply:
This is exactly what TG has, but I will remove it if its bad

Reply:
Very very old implementation of SS's, well before SS defines became a thing.